plugins/cpp_metrics/parser/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ include_directories(
   ${PLUGIN_DIR}/model/include)
 
 add_library(cxxmetricsparser SHARED
+  src/efferent.cpp
```

Comment:
`efferent.h` should also be added here, so CMake can recognize that in case the header file changes, recompilation of the target is required.